### PR TITLE
*: Support show background DDL info

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: executor/executor.go [hunk 1]
```diff
@@ -115,22 +115,37 @@ func (e *ShowDDLExec) Next() (*Row, error) {
...
 	}
-	if info.Job != nil {
-		rowData[2] = info.Job.String()
+	if bgInfo.Owner != nil {
+		rowData[4] = bgInfo.Owner.String()
 	}
+	if bgInfo.Job != nil {
+		rowData[5] = bgInfo.Job.String()
+	}
+
```
Comment (coocood) [inline executor/executor.go]: Don't use hard coded '1', '2', '4', '5', declare a variable before rowData, assign its value, then initialzie the rowData.
It makes it easier to change the order of the fields or add more fields in the future.

like
`
rowData := []interface{}{
		ddlInfo.SchemaVer,
		ddlOwner,
		ddlJob,
		bgInfo.SchemaVer,
		bgOwner,
		bgJob,
	}
`

File: optimizer/plan/planbuilder.go [hunk 2]
```diff
@@ -511,10 +511,13 @@ func (b *planBuilder) buildAdmin(as *ast.AdminStmt) Plan {
...
 
 func buildShowDDLFields() []*ast.ResultField {
-	rfs := make([]*ast.ResultField, 0, 3)
+	rfs := make([]*ast.ResultField, 0, 6)
 	rfs = append(rfs, buildResultField("", "SCHEMA_VER", mysql.TypeLonglong, 4))
 	rfs = append(rfs, buildResultField("", "OWNER", mysql.TypeVarchar, 64))
 	rfs = append(rfs, buildResultField("", "Job", mysql.TypeVarchar, 128))
+	rfs = append(rfs, buildResultField("", "BG_SCHEMA_VER", mysql.TypeLonglong, 4))
+	rfs = append(rfs, buildResultField("", "BG_OWNER", mysql.TypeVarchar, 64))
+	rfs = append(rfs, buildResultField("", "BG_Job", mysql.TypeVarchar, 128))
```
Comment (qiuyesuifeng) [inline optimizer/plan/planbuilder.go]: Why not using BG_JOB?

**Comment by zimulala** [comment]
PTAL @coocood @qiuyesuifeng

**Comment by coocood** [comment]
LGTM

File: inspectkv/inspectkv.go [hunk 3]
```diff
@@ -67,8 +67,8 @@ func GetDDLInfo(txn kv.Transaction) (*DDLInfo, error) {
 	return info, nil
 }
 
-// GetDDLBgInfo returns DDL background information.
-func GetDDLBgInfo(txn kv.Transaction) (*DDLInfo, error) {
+// GetBgDDLInfo returns DDL background information.
+func GetBgDDLInfo(txn kv.Transaction) (*DDLInfo, error) {
```
Comment (qiuyesuifeng) [inline inspectkv/inspectkv.go]: A little confused why rename GetDDLBgInfo to GetBgDDLInfo,  from your comment `DDL background information`, seems GetDDLBgInfo is clearer.

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**